doc/debugging-an-app.md

```diff
@@ -29,15 +34,17 @@ web           1/1    94.92%   2.0rps           5ms          10ms          18ms
 We can see that the `voting` service is performing far worse than the others.
 
 How do we figure out what's going on? Our traditional options are: looking at
-the logs, attaching a debugger, etc. Conduit gives us a new tool that we can use
-- a live view of traffic going through the deployment. Let's use the `tap`
+the logs, attaching a debugger, etc. Linkerd gives us a new tool that we can use
+: a live view of traffic going through the deployment. Let's use the `tap`
 command to take a look at all requests currently flowing to this deployment.
```

Comment:
I'm not entirely sure why, but this paragraph doesn't render properly as a result of this change:

![image](https://user-images.githubusercontent.com/9226/42847371-8d8a761e-89d0-11e8-9ed3-2841c8796317.png)

I think it's due to the `:` at the start of line 38. Changing to this fixes it:

```
How do we figure out what's going on? Our traditional options are: looking at
the logs, attaching a debugger, etc. Linkerd gives us a new tool that we can
use: a live view of traffic going through the deployment. Let's use the `tap`
command to take a look at all requests currently flowing to this deployment.
```